### PR TITLE
Add neighbours API

### PR DESCRIPTION
**Pull request by mxpv**
This PR adds neighbours API (`ip neighbour show/add/delete`)

Signed-off-by: Maksym Pavlenko <pavlenko.maksym@gmail.com>

### CODE REVIEW

File: rtnetlink/src/neighbour/add.rs [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+        message.nlas.push(Nla::Destination(match destination {
+            IpAddr::V4(v4) => v4.octets().to_vec(),
+            IpAddr::V6(v6) => v6.octets().to_vec(),
+        }));
+
+        NeighbourAddRequest { handle, message }
+    }
+
+    pub fn state(mut self, state: u16) -> Self {
```
Comment (little-dude) [inline rtnetlink/src/neighbour/add.rs:41]: We should explain what the `state`, `flags` and `ntype` are about. This is already documented a little in `netlink-packet-route`: https://docs.rs/netlink-packet-route/0.6.0/netlink_packet_route/rtnl/struct.NeighbourHeader.html.

**Review by little-dude — APPROVED** [review]
Thanks for the PR @mxpv. It looks good, I'd just like to have a bit more documentation.

**Comment by mxpv** [comment]
@little-dude thanks for looking into this! I've pushed a commit with docs.

**Comment by little-dude** [comment]
Thanks @mxpv! I'll keep this open a couple days in case someone else want to review.

**Comment by little-dude** [comment]
Thank you!

**Pull request merged**